packages/x-date-pickers/src/locales/itIT.ts

```diff
@@ -64,17 +64,17 @@ const itITPickers: Partial<PickersLocaleText<any>> = {
       : "Scegli l'ora",
 
   // Table labels
-  timeTableLabel: 'scegli un ora',
+  timeTableLabel: "scegli un'ora",
   dateTableLabel: 'scegli una data',
 
   // Field section placeholders
-  // fieldYearPlaceholder: params => 'Y'.repeat(params.digitAmount),
-  // fieldMonthPlaceholder: params => params.contentType === 'letter' ? 'MMMM' : 'MM',
-  // fieldDayPlaceholder: () => 'DD',
-  // fieldHoursPlaceholder: () => 'hh',
-  // fieldMinutesPlaceholder: () => 'mm',
-  // fieldSecondsPlaceholder: () => 'ss',
-  // fieldMeridiemPlaceholder: () => 'aa',
+  fieldYearPlaceholder: (params) => 'a'.repeat(params.digitAmount),
```

Comment:
It is intended to have lower case letter for year / day and higher case letter for month ?
You would have formats like `gg/MM/aaaa`

Reply:
You are right. It doesn't change anything in terms of meaning, so to keep consistency with other locales, it is better to capitalize year and day.